### PR TITLE
Xmake-io.Xmake version 2.5.5

### PR DESCRIPTION
**Pull request by quhxl**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19660)

### CODE REVIEW

File: manifests/x/Xmake-io/Xmake/2.5.5/Xmake-io.Xmake.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Xmake-io.Xmake
+PackageVersion: 2.5.5
```
Comment (ItzLevvie) [inline manifests/x/Xmake-io/Xmake/2.5.5/Xmake-io.Xmake.installer.yaml:5]:
```suggestion
PackageVersion: 2.5.5+202107030400
```

File: manifests/x/Xmake-io/Xmake/2.5.5/Xmake-io.Xmake.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Xmake-io.Xmake
+PackageVersion: 2.5.5
```
Comment (ItzLevvie) [inline manifests/x/Xmake-io/Xmake/2.5.5/Xmake-io.Xmake.yaml:5]:
```suggestion
PackageVersion: 2.5.5+202107030400
```

File: manifests/x/Xmake-io/Xmake/2.5.5/Xmake-io.Xmake.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: Xmake-io.Xmake
+PackageVersion: 2.5.5
```
Comment (ItzLevvie) [inline manifests/x/Xmake-io/Xmake/2.5.5/Xmake-io.Xmake.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.5.5+202107030400
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by quhxl** [comment]
@ItzLevvie Why is this happening now?!

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You'll have to update the version folder using GitHub Desktop.

**Comment by quhxl** [comment]
> 
> 
> You'll have to update the version folder using GitHub Desktop.

I could have thought of that myself. Thanks!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**